ecs-init/docker/docker_test.go

```diff
@@ -229,6 +229,7 @@ func validateCommonCreateContainerOptions(opts godocker.CreateContainerOptions,
 	expectKey("ECS_ENABLE_TASK_IAM_ROLE=true", envVariables, t)
 	expectKey("ECS_ENABLE_TASK_IAM_ROLE_NETWORK_HOST=true", envVariables, t)
 	expectKey("ECS_ENABLE_TASK_ENI=true", envVariables, t)
+	expectKey("ECS_ENABLE_AWSLOGS_EXECUTIONROLE_OVERRIDE=true", envVariables, t)
```

Comment:
We should have a test that verifies that this is *not* set when not running Amazon Linux.

Reply:
> a test that verifies that this is not set when not running Amazon Linux.

sure. what would be the best way to test that?

Reply:
tracking in #131